### PR TITLE
Feature/add folder

### PR DESCRIPTION
**Pull request by sher009**
Create a public class Author with next private fields:
    
    
    private String firstName;
    private String lastName;
    private String birthDate;
    private String language;

### CODE REVIEW

File: src/main/java/com/example/bookstore/model/Author.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.example.bookstore.model;
+
+public class Author {
```
Comment (kostyaskrun) [inline src/main/java/com/example/bookstore/model/Author.java]: can you add private fields as well in this PR?

Comment (sher009) [reply src/main/java/com/example/bookstore/model/Author.java]: I need to add private fields in the PR, without changing anything else in the code. Is that correct?

Comment (kostyaskrun) [reply src/main/java/com/example/bookstore/model/Author.java]: Yeah, exactly. You need to add attributes to this class. i.e. Name, birthdate, country, language and others.

Comment (sher009) [reply src/main/java/com/example/bookstore/model/Author.java]: I dont understud all, can You check now. And I dont have in code country, to add it?

File: src/main/java/com/example/bookstore/model/Author.java [hunk 2]
```diff
@@ -1,4 +1,57 @@
 package com.example.bookstore.model;
 
-public class Author {
+public class Author{
+    // Поля класу; class attributes
+    private String firstName;
+    private String lastName;
+    private String birthDate;
```
Comment (kostyaskrun) [inline src/main/java/com/example/bookstore/model/Author.java]: You did a great job! But I have one small remark about birthDate field. Can you replace it with LocalDate class?

**Pull request merged**